cmd/bootstrap/runner.go

```diff
@@ -538,10 +577,12 @@ func (r *runner) installOperator(ctx context.Context, helmClient helmclient.Inte
 			return microerror.Mask(err)
 		}
 
-		// ReleaseName has unique suffix like in the control plane so the test
-		// app CRs need to use 0.0.0 for the version label.
 		opts := helmclient.InstallOptions{
+			// ReleaseName has unique suffix like in the control plane so the test
+			// app CRs need to use 0.0.0 for the version label.
 			ReleaseName: fmt.Sprintf("%s-unique", name),
+			// Wait ensures that the status webhook is ready.
+			Wait: true,
```

Comment:
@piontec Adding the wait flag for ready deployments makes things more reliable. Thanks for the hint. ❤️